.env.example

```diff
@@ -202,5 +215,12 @@ VITE_APP_TITLE="LibreChat"
 # and have set the GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET above
 VITE_SHOW_GOOGLE_LOGIN_OPTION=false
 
+# Enable OpenID Login
+# This enables/disables the Login with OpenID button on the login page.
+# Set to true if you have registered the app with an OpenID provider
+# and have set the OPENID_CLIENT_ID, OPENID_CLIENT_SECRET, OPENID_ISSUER, OPENID_SESSION_SECRET, and OPENID_SCOPE in the /api/.env file
+VITE_SHOW_OPENID_LOGIN_OPTION=false
```

Comment:
Minor one, but could you change this to `ALLOW_OPENID` or maybe `ALLOW_OPENID_LOGIN`? 
I want to change the google one too (in the future) but anything with the prefix `ALLOW_` will be passed to vite and it saves changing it in the future if we stop using vite. 

Side note: shorter env var names is something I'm pushing for a little more in projects as certain platforms (ie. AWS lambda) have a limited number of characters in `.env` files